package.json

```diff
@@ -35,7 +36,6 @@
     "release": "np --no-yarn && git push https://github.com/terrestris/geostyler.git master --tags",
     "styleguide": "styleguidist server",
     "start:styleguide": "styleguidist server",
-    "start:dev": "webpack --mode=development --watch --config dev-build.config.js",
```

Comment:
Is there another way to have a dev build continually built? This is really handy when developing with non es6 projects.

Reply:
I was quite sure that this was an old script so i removed it... I'll have a look at it, and bring it back maybe.